gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -243,6 +251,9 @@ class {{ service.async_client_name }}:
 
         )
 
+        if CLIENT_LOGGING_SUPPORTED:
```

Comment:
I don't think we should have an extra line at each logging site to check the logging enablement. Rather, I suggest something like what I propose in prototype R-0 in in [go/cloudsdk-python-logging-performant](http://goto.google.com/cloudsdk-python-logging-performant), to abstract away any checking we may want to do while providing an interface (only used by us, not our users) that is parallel to the standard loggers.